doc/man/cockpit.conf.xml

```diff
@@ -74,6 +74,42 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
     </variablelist>
   </refsect1>
 
+  <refsect1 id="oauth">
+	  <title>oauth</title>
```

Comment:
All the auth stuff uses lower case. bearer, basic etc...

But i can switch to camel for this if you think that's better.